### PR TITLE
Coding: Develop First Time Issue Section of Contribute Page

### PR DESCRIPTION
**Pull request by rutvi18**
### Description

Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes # [ISSUE]

### Type of Change:
**Delete irrelevant options.**

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
If you are working on a design then add a gif to show the responsiveness of your design.
Please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR) and share the link of deployed site here.

### Checklist:
**Delete irrelevant options.**

- [x]  My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by rutvi18** [comment]
There is some issue with my code in contibuteintro.js. I would like to get the feedback regarding the same.

**Comment by tichnas** [comment]
Hi @rutvi18, thanks for this PR.
Please update the main comment to include a description of what you did, the issue number you solved and delete the irrelevant options under `Type of Change`. Also, to check some points, you write `[x]` (instead of `[1 ]`), you can refer to other PRs also.

**Comment by tichnas** [comment]
It looks like there is no change in `contibuteintro.js`, please push the changes so that everyone can help with any issue you're facing.

**Comment by rutvi18** [comment]
@tichnas Ok. Thanks for feedback

**Comment by rutvi18** [comment]
Screenshot (83)

This error is being displayed. Please help for the debugging the same.

File: src/content/contributeintro.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+              'dfbhsdjs',
+          },
+          {
+            par:
+              "jhgfjgrrj",
+          },
+        ],
+      },
+    ],
+} ,
```
Comment (tichnas) [inline src/content/contributeintro.js:40]: This should be `} ;` instead of `} ,`

**Comment by tichnas** [comment]
Please try after doing the change I suggested.
Also, I think you already opened another PR for the same issue (#160)

**Pull request closed without merging**